web/modules/custom/owner/owner.install

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * @file
+ * Install, uninstall, schema and requirements hook for the owner module.
+ */
+
+/**
+ * Implements hook_update_N().
+ *
+ * Fix the owner type predicate.
+ */
+function owner_update_8100() {
+  /** @var \Drupal\Core\Database\Connection $query */
+  $query = \Drupal::service('sparql_endpoint');
```

Comment:
I saw some other remarks where it was asked to use a better variable for $query, like $sparql connection or similar. Maybe we could uniform more?